marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/MappingControllerTest.java

```diff
@@ -51,7 +51,7 @@ void testValidateMappings() throws Exception {
             .andDo(response -> {
                 JsonNode errorResult = parseJsonResponse(response);
                 assertEquals("concat(name, ')", errorResult.get("properties").get("name").get("sourcedFrom").asText());
-                assertEquals("Invalid XPath expression: concat(name, ')", errorResult.get("properties").get("name").get("errorMessage").asText());
+                assertEquals("Invalid XPath Expression: 'concat(name, ')'. Cause: Unexpected character.", errorResult.get("properties").get("name").get("errorMessage").asText());
```

Comment:
I know this is picky - but I don't think we should capitalize "Expression" here per the story; it's not a proper noun, and it'd be inconsistent with dozens of other messages that don't capitalize non-proper nouns.